acvm-repo/acvm/src/pwg/brillig.rs

```diff
@@ -206,18 +214,14 @@ impl<'b, B: BlackBoxFunctionSolver<F>, F: AcirField> BrilligSolver<'b, F, B> {
                                     }))
                                 }
                             }
```

Comment:
if it doesn't throw you into borrow checker hell, maybe it's a good idea to move the left hand side of `let payload = ...` to a function, for readability